iphone/Classes/TiGradient.m

```diff
@@ -129,16 +133,29 @@ - (void)setEndPoint:(id)newEnd
   } else {
     [endPoint setValues:newEnd];
   }
+  [self replaceValue:newEnd forKey:@"endPoint" notification:NO];
 }
 
 - (void)setStartRadius:(id)newRadius
 {
   startRadius = [TiUtils dimensionValue:newRadius];
+  [self replaceValue:newRadius forKey:@"startRadius" notification:NO];
 }
 
 - (void)setEndRadius:(id)newRadius
 {
   endRadius = [TiUtils dimensionValue:newRadius];
+  [self replaceValue:newRadius forKey:@"endRadius" notification:NO];
+}
+
+- (NSNumber *)endRadius
+{
+  return [self valueForUndefinedKey:@"endRadius"];
+}
+
+- (NSNumber *)startRadius
+{
+  return [self valueForUndefinedKey:@"startRadius"];
```

Comment:
We don't document percentage based radius settings, but this will return it as a number between 0-100, right? I think it should return the original percentage string.

If we don't officially support percentage based radius, then I'm okay for making this a task for later.

Reply:
See comment above. But you are right, I need to change the return type to `id`.